wiki/Beatmaps/en.md

```diff
@@ -3,57 +3,153 @@
 <!-- TODO:
 - figure out what the right relationship is between this and the Beatmapsets article, it doesn't make a lot of sense right now because most of this info can be about beatmapsets too
 - maybe the ranked statuses / beatmap categories should be its own article?
-- relationship to beatmap packs, osu!direct, charts, leaderboards, contests, ...
 -->
 
 *Not to be confused with [Difficulties](/wiki/Difficulties).*
 
-**Beatmaps** are game levels created for songs. They contain [hit objects](/wiki/Hit_Objects), [timing](/wiki/Beatmap_Editor/Timing)<!-- TODO: this should not be referencing the editor --> information, difficulty settings, and visual effects<!-- TODO: more links here -->.
+**Beatmaps** are game levels created for songs and stored in text files with [the `.osu` extension](/wiki/osu!_File_Formats/Osu_(file_format)). They have different structure and sometimes may be played only in one of osu!'s [game modes](/wiki/Game_mode).
 
-Beatmaps are made as part of **[beatmapsets](Beatmapsets)**. Beatmapsets group beatmaps together so that they can be [submitted](/wiki/Glossary#bss) to the website and show up on the [beatmap listing](https://osu.ppy.sh/beatmapsets). They typically cover one song, where each beatmap of the set is a different [difficulty](/wiki/Difficulties).
+## Overview
+
+At the core, a beatmap consists of [hit objects](/wiki/Hit_object) of a certain [game mode](/wiki/Game_mode), which are set up according to [timing](/wiki/Glossary#timing-section) tailored to a specific song. Beatmaps have different aspects, which directly affect user experience.
+
+The hit objects of a beatmap compose [patterns](Pattern), which also vary per game mode and set the tone for overall difficulty, and how a beatmap should be played. The process of playing the beatmap is altered by various [difficulty settings](/wiki/Beatmap_Editor/Song_Setup#difficulty) used by the beatmap's author and [game modifiers](/wiki/Game_modifier) toggled by a player.
+
+From the auditory perspective, a beatmap can augment the song by providing one or multiple sets of audio samples, or [hitsounds](/wiki/Beatmapping/Hitsound), that serve as an additional layer of feedback.
+
+Visual components of a beatmap, contrary to its patterns, provide auxiliary experience during gameplay and can be disabled in the [visual settings](/wiki/Visual_Settings) popup for the most part. They include:
+
+- [Background imagery](Background) and video, which both can act as a playfield;
+- [Kiai time](/wiki/Kiai_time) and [combo colours](/wiki/Glossary/Combo_colour) that help distinguish different beatmap sections from each other;
+- [Storyboard](/wiki/Storyboards) that adds motion graphics and special effects, or provides a background story or theme for the song;
+- [Custom skin](/wiki/Skinning), which changes the appearance of interface and gameplay elements;
+- [Title text](Title_Text), which presents the song's metadata.
```

Comment:
```suggestion
- [background imagery](Background) and video, which both can act as a playfield;
- [Kiai time](/wiki/Kiai_time) and [combo colours](/wiki/Glossary/Combo_colour) that help distinguish different beatmap sections from each other;
- [storyboard](/wiki/Storyboards) that adds motion graphics and special effects, or provides a background story or theme for the song;
- [custom skin](/wiki/Skinning), which changes the appearance of interface and gameplay elements;
- [title text](Title_Text), which presents the song's metadata.
```

Reply:
shouldn't these be in sentence case? aldo, why is kiai time left out, is it a proper noun?

Reply:
i think they shouldnt be capitalized as the list is placed after a colon suggesting enumeration

about kiai, i thought it was a proper noun but it may actually be decapitalized i guess ¯\_(ツ)_/¯